src/scripts/main.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', event => {
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+
+  if (event.target !== wall) {
+    return;
+  }
+
+  const posX = event.offsetX - spider.width / 2;
+  const posY = event.offsetY - spider.height / 2;
+  const getPosition = (coord, wallSize, spiderSize) => {
+    return (coord < 0)
+      ? 0
+      : (coord > wallSize - spiderSize)
+        ? wallSize - spiderSize
+        : coord;
+  };
```

Comment:
for the sake of readability, it's better to rewrite this condition with if-else statements